addons/csw/functions/fnc_reload_getLoadableMagazines.sqf

```diff
@@ -0,0 +1,52 @@
+/*
+ * Author: PabstMirror
+ * Gets magazines that the player is carrying that can be loaded into the static weapon
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ * 1: Player <OBJECT>
+ *
+ * Return Value:
+ * Mags <ARRAY>
+ * [Carry Magazine <STRING>, Turret Path <ARRAY>, Ammo Needed <NUMBER>]
+ *
+ * Example:
+ * [cursorObject, player] call ace_csw_fnc_reload_getLoadableMagazines
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_vehicle", "_player"];
+
+private _carriedMagazines = [];
+
+{
+    if (isClass (configFile >> QGVAR(groups) >> _x)) then {
+        _carriedMagazines pushBackUnique _x;
+    };
+} forEach (magazines _player);
```

Comment:
Would be nice to see ability to load magazines from groundWeaponholder.

Reply:
This PR is for all intents and purposes feature complete